src/moveit_visual_tools.cpp

```diff
@@ -1605,10 +1611,10 @@ planning_scene_monitor::PlanningSceneMonitorPtr MoveItVisualTools::getPlanningSc
 {
   if (!psm_)
   {
-    ROS_INFO_STREAM_NAMED(LOGNAME, "No planning scene passed into moveit_visual_tools, creating one.");
+    RCLCPP_INFO_STREAM(LOGGER, "No planning scene passed into moveit_visual_tools, creating one.");
     loadPlanningSceneMonitor();
-    ros::spinOnce();
-    ros::Duration(1).sleep();  // TODO: is this necessary?
+    rclcpp::spin_some(node_);
+    rclcpp::sleep_for(std::chrono::milliseconds(1000));  // TODO: is this necessary?
```

Comment:
```suggestion
    rclcpp::sleep_for(1s);  // TODO: is this necessary?
```
using chrono literals.
Also, it should really not be necessary, `loadPlanningSceneMonitor()` should provide a fully initialized PSM

Reply:
This is removed as spin_some was not necessary.